### PR TITLE
[full-ci] Purge confirmation modal for deletion from files list

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes https://github.com/owncloud/web/issues/5974

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by JanAckermann** [comment]
Success notifications are missing

File: packages/web-app-files/src/composables/actions/files/useFileActionsDelete.ts [hunk 1]
```diff
@@ -14,26 +14,31 @@ import {
...
   const router = useRouter()
   const hasSpaces = useCapabilitySpacesEnabled()
   const hasPermanentDeletion = useCapabilityFilesPermanentDeletion()
-  const { displayDialog } = useFileActionsDeleteResources({ store })
+  const { displayDialog, filesList_delete } = useFileActionsDeleteResources({ store })
 
   const { $gettext } = useGettext()
 
-  const handler = ({ space, resources }: FileActionOptions) => {
+  const handler = ({ space, resources, deletePermanent }) => {
```
Comment (dschmidt) [inline packages/web-app-files/src/composables/actions/files/useFileActionsDelete.ts:29]:
```suggestion
const handler = ({ space, resources, deletePermanent }: FileActionOptions & { deletePermanent: boolean }) => {```

Comment (JanAckermann) [reply packages/web-app-files/src/composables/actions/files/useFileActionsDelete.ts]: :+1:

File: packages/web-app-files/src/store/actions.ts [hunk 2]
```diff
@@ -200,7 +200,7 @@ export default {
 
             translated = $gettext('Failed to delete "%{file}" - the file is locked')
           }
-          const title = $gettextInterpolate(translated, { file: file.name }, true)
+          const title = $gettext(translated, { file: file.name }, true)
```
Comment (JammingBen) [inline packages/web-app-files/src/store/actions.ts:203]: You can remove `$gettext` from `translated`, otherwise you're wrapping multiple `$gettext` calls.

**Review by JammingBen — APPROVED** [review]

**Pull request merged**